docker/Dockerfile

```diff
@@ -35,6 +43,8 @@ FROM base
 ARG BUILD
 ARG GITHUB_SHA
 COPY --from=builder /build /
+COPY --from=go-builder /app/whep_proxy /usr/local/bin/whep_proxy
+RUN chmod +x /usr/local/bin/whep_proxy
 ENV PYTHONUNBUFFERED=1 FLASK_APP=frontend BUILD=$BUILD GITHUB_SHA=$GITHUB_SHA
 WORKDIR /app
 CMD ["flask", "run", "--host=0.0.0.0"]
```

Comment:
Could we run the Go proxy and the python app in parallel? This may alleviate some added latency when starting a stream with these new cams.